### PR TITLE
Help, we're almost there...

### PR DESCRIPTION
**Pull request by johnnyhalife**
I think I nailed most of it, the problem is that either Windows Azure is not working properly or I'm doing something wrong. Probably both, but since my Windows Machine died and I cannot TS into the VM please help me wrap this up.

:metal: awesome project btw :metal:

### CODE REVIEW

**Comment by tjanczuk** [comment]
I think one extra change that might be needed now that instead of `git.exe` we are calling `git.cmd` is changing all lines that call `%GIT% something-somenting` to `call %GIT% something-something`. Specifically:

https://github.com/tjanczuk/git-azure/blob/master/src/bootstrap/setup_worker.cmd#L131
https://github.com/tjanczuk/git-azure/blob/master/src/bootstrap/setup_worker.cmd#L140
https://github.com/tjanczuk/git-azure/blob/master/src/bootstrap/sync_repo.cmd#L11
https://github.com/tjanczuk/git-azure/blob/master/src/bootstrap/sync_repo.cmd#L20
https://github.com/tjanczuk/git-azure/blob/master/src/bootstrap/sync_repo.cmd#L29

Could you give it a shot? I am stuck prepping for two important presentations of this stuff today and tomorrow, I'd rather not take unvalidated pool request in at this point. Friday I will be open to experimenting.

**Comment by johnnyhalife** [comment]
I'll build the new cpskg and give it another shot

**Comment by johnnyhalife** [comment]
Still no luck :disappointed:, you might have a better shot. Let I'm trying
to build a VM or something where I can debug but right now it's stucked on
"BUSY"

On Wed, Jun 20, 2012 at 6:16 PM, Tomasz Janczuk <
reply@reply.github.com

**Comment by tjanczuk** [comment]
At this point you should be able to RDP to the machine and you can inspect e:\approot\log.txt. 

Barring that I can look at this tomorrow.

File: src/bootstrap/setup_worker.cmd [hunk 1]
```diff
@@ -19,10 +20,10 @@ if %ERRORLEVEL% NEQ 0 (
 )
 echo %DATE% %TIME% Permissions granted
 
-if exist %THIS%\node.msi if exist %THIS%\minigit.zip if exist %THIS%\freesshd.exe goto install_node
+if exist %THIS%\node.msi if exist %THIS%\Git-1.7.9-preview20120201.exe if exist %THIS%\freesshd.exe goto install_node
```
Comment (tjanczuk) [inline src/bootstrap/setup_worker.cmd:23]: This needs to change to the new Portable git ZIP, but let me take care of it after I merge.

**Pull request merged**